### PR TITLE
VACMS-14600 add tugboat for PR environments

### PR DESCRIPTION
**Pull request by tjheffner**
## Description
Closes [#14600](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14600)

## Testing done
tinkered with settings in `.tugboat/config.yml` and our `drupalClient.ts` until pages properly built inside a tugboat container.

## Screenshots
Screenshot 2023-08-11 at 12 04 54 PM

Also, just look down below in this PR's comments 😄 

## QA steps

Locally, the site should:
- still build as expected when pointing at a local cms endpoint using `yarn export` & `yarn dev`
- still build as expected when pointing at the content build endpoint using `yarn export` & `yarn dev`

Tugboat build should succeed without failing:
Screenshot 2023-08-11 at 12 11 03 PM

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### CODE REVIEW

**Comment by tjheffner** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr138-m9rg211wvcaj7annfqxi0q01wup31tph.tugboat.vfs.va.gov


Dashboard:
- https://tugboat.vfs.va.gov/64d556362d3036648da7eb12

**Comment by tjheffner** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr138-m9rg211wvcaj7annfqxi0q01wup31tph.tugboat.vfs.va.gov


Dashboard:
- https://tugboat.vfs.va.gov/64d556362d3036648da7eb12

File: .tugboat/config.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+services:
+  # Configure a standard apache webserver to host our site.
+  apache:
```
Comment (timcosgrove) [inline .tugboat/config.yml:3]: Do we generally use Apache? Not objecting, but would normally expect nginx

**Review by timcosgrove — APPROVED** [review]
Just a question FMSA

File: .tugboat/config.yml (hunk 1 above)
Comment (tjheffner) [reply .tugboat/config.yml:3]: I was referencing https://docs.tugboatqa.com/starter-configs/examples/static-html/ & https://docs.tugboatqa.com/starter-configs/examples/gatsby/ which are both using the `tugboatqa/httpd` image and `apache`. we can definitely change this if needed

**Pull request merged**